olwebsite-app/src/pages/RegisterView.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import Input from '../components/Objects/Input/Input';
+import homeLogo from "../images/homeLogo.png";
+
+export default function RegisterView() {
+  /************************************
+   * Render
+   ************************************/
+
+  return (
+    <div className="register-view-body ">
+        <div className="register-input-container"> 
+            <img src={homeLogo} alt="oneleif logo" />
+            <form>
+                <Input className={'auth'} label={'Email'}/>
+                <Input className={'auth'} label={'Password'} type={"password"}/>
+                <Input className={'auth'} label={'Reenter Password'} type={"password"}/>
+                <input type="submit" value="Sign up" />
```

Comment:
Why an input instead of a button? Despite being functionally interchangeable, I think a button might be best since it's easier to customize and more easily matched when testing. We're probably going to be overriding the default action for submitting as well.

Reply:
Sounds good to me